pom.xml

```diff
@@ -174,6 +174,18 @@
         <version>23.4.0.24.05</version>
         <scope>test</scope>
       </dependency>
+      <dependency>
+        <groupId>org.testcontainers</groupId>
+        <artifactId>mssqlserver</artifactId>
+        <version>${testcontainers.version}</version>
+        <scope>test</scope>
+      </dependency>
+      <dependency>
+        <groupId>com.microsoft.sqlserver</groupId>
+        <artifactId>mssql-jdbc</artifactId>
+        <version>12.6.1.jre11</version>
+        <scope>test</scope>
+      </dependency>
     </dependencies>
   </dependencyManagement>
   <build>
```

Comment:
We should push all these MS SQL tests out to their own CI builds.  Could you create an additional profile in `pom.xml` for this? Example for postgres
```
    <profile>
      <id>only-postgres-tests</id>
      <build>
        <plugins>
          <plugin>
            <artifactId>maven-surefire-plugin</artifactId>
            <configuration>
              <includes>
                <include>**/*Postgres*.java</include>
              </includes>
            </configuration>
          </plugin>
        </plugins>
      </build>
    </profile>
```

And then add it to the build matrix here: https://github.com/gruelbox/transaction-outbox/blob/master/.github/workflows/pull_request.yml#L33

THe idea is that the tests all use H2 only unless one or more of these profiles are enabled.

Reply:
Should be addressed but let me know if I missed something.